pygeo/mphys/mphys_dvgeo.py

```diff
@@ -15,7 +15,7 @@ def initialize(self):
 
         self.options.declare("ffd_file", default=None)
         self.options.declare("vsp_file", default=None)
-        self.options.declare("vsp_options", default=None)
+        self.options.declare("vsp_options", default={})
```

Comment:
I think we shouldn't make that default argument `{}`.  Since it is a mutable type, if we have more than one DVGeoVSP objects by default the `vsp_options` objects will be linked. 
Perhaps we could make the option required and add in the description that it is dictionary of options and remove the default altogether.

Reply:
Agreed. I changed the default to a None. Later in the setup procedure (https://github.com/timryanb/pygeo/blob/vsp-volume-proj/pygeo/mphys/mphys_dvgeo.py#L27-L34), I check if the value is None and if so pass a unique empty dict for each DVGeo instance